tests/types/test_uuid.py

```diff
@@ -32,7 +41,19 @@ def test_uuid_load(conn, fmt_out):
         copy.set_types(["uuid"])
         (res,) = copy.read_row()
 
-    assert res == UUID(val)
+    uuid_val = UUID(val)
+    assert res == uuid_val
+    # the C modules bypasses __init__, so checking the state of the UUID object
+    assert res.hex == uuid_val.hex
+    assert res.int == uuid_val.int
+    assert res.bytes == uuid_val.bytes
+    assert res.is_safe == uuid_val.is_safe
+    assert getattr(UUID, "__slots__") == (
```

Comment:
What is wrong with `UUID.__slots__`? :slightly_smiling_face:

Reply:
mypy complaints about it

<img width="665" alt="Screenshot 2025-02-08 at 08 41 22" src="https://github.com/user-attachments/assets/5bf581f9-49cc-42cd-9e6a-bffa9f43565f" />

Reply:
Looks like `__slots__` aren't really present much in typeshed, here's [an open ticket](https://github.com/python/typeshed/issues/8832). Will update with a type ignore.